addons/mail/tests/test_mail_channel.py

```diff
@@ -290,6 +314,19 @@ def test_channel_info_get(self):
         same_solo_channel_info = self.env['mail.channel'].channel_get(partners_to=self.partner_employee_nomail.ids)
         self.assertEqual(same_solo_channel_info['id'], solo_channel_info['id'])
 
+    # `channel_get` will pin the channel by default and thus last meaningful action time will be updated.
```

Comment:
Is it because it is pinning it by default? Isn't it because the default value is now on creation?

Reply:
And so there is a missing test for the flux of actually pinning it.